ddtrace/internal/buffer.py

```diff
@@ -0,0 +1,57 @@
+from collections import deque
+import threading
+
+from ddtrace.vendor import attr
+
+
+class BufferFull(Exception):
+    pass
+
+
+class BufferItemTooLarge(Exception):
+    pass
+
+
+@attr.s
+class TraceBuffer(object):
```

Comment:
Would be great to have some doc / comment.

Reply:
Added! Let me know what you think